controller/workitem.go

```diff
@@ -605,6 +611,26 @@ func ConvertWorkItem(request *goa.RequestData, wi workitem.WorkItem, additional
 	return op
 }
 
+// WorkItemIncludeHasChildren adds meta information about existing children
+func WorkItemIncludeHasChildren(appl application.Application, ctx context.Context) WorkItemConvertFunc {
```

Comment:
Out of curiosity, do you need to export the function ? (naming with a capital letter)